seqr/views/utils/anvil_metadata_utils.py

```diff
@@ -121,7 +120,12 @@ def _format_transcript_id(transcript_id, transcript):
 }
 
 
-def _get_family_metadata(family_filter, family_fields, include_metadata, include_mondo, format_id):
+def _get_family_metadata(family_filter, family_fields, include_family_name_display, include_family_sample_metadata, include_mondo, format_id):
+    family_fields = {'analysis_groups': {
+        'value': ArrayAgg('analysisgroup__name', distinct=True, filter=Q(analysisgroup__isnull=False)),
+        'format': lambda f: '; '.join(f['analysis_groups']),
+    }} if include_family_sample_metadata else family_fields
+    include_family_name_display = include_family_name_display or include_family_sample_metadata
```

Comment:
Why are we using `include_family_sample_metadata` to reset the value of `include_family_name_display`? Can we instead be more explicit in` setting include_family_name_display=True` when we want `FAMILY_NAME_DISPLAY_VALUES`?

Reply:
because the "family_sample_metadata" includes the display name fields, so setting `include_family_name_display` to true implies `include_family_name_display`.  It seemed messier to require multiple function to pass a redundant extra flag than to set the behavior explicitly here